wear/build-sdk.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
Is it a duplicate from the `app/build-sdk.sh` file? If so, there is no need to do that.